### PR TITLE
Handle viewer calls

### PR DESCRIPTION
**Pull request by fjvallarino**
@tejohnso @ahmedalsudani please review

### CODE REVIEW

**Comment by tejohnso** [comment]
:+1:

File: installer/launcher.js [hunk 1]
```diff
@@ -32,13 +21,12 @@ module.exports = {
     })
     .then(()=>{
       log.all("cache start", "", "50%");
-      startCache();
+      module.exports.startCache();
```
Comment (ahmedalsudani) [inline installer/launcher.js]: @fjvallarino Is it necessary to call `startCache` as a child of `module.exports`?

**Comment by fjvallarino** [comment]
@ahmedalsudani I did it that way so it can be mocked on unit tests; otherwise it will always call the internal version and try to start a real Java process.

**Comment by ahmedalsudani** [comment]
@fjvallarino I see. Looks good then. I found 4 lint errors that have probably been there even before this PR, but do you mind fixing them?

```
installer/config.js: line 40, col 32, 'reject' is defined but never used.
installer/config.js: line 97, col 32, 'reject' is defined but never used.
installer/config.js: line 2, col 1, 'network' is defined but never used.

installer/edit-config.js: line 3, col 1, 'ws' is defined but never used.
```

Aside from that :+1:

**Comment by tejohnso** [comment]
@fjvallarino I think this can be closed

**Comment by fjvallarino** [comment]
@tejohnso yeah, not much is lost, and bringing it up to date would be harder than rewriting.

**Pull request closed without merging**